plugin/action/throttle/throttle.go

```diff
@@ -4,6 +4,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/go-redis/redis"
```

Comment:
you created the redisClient interface, but you still depend on the go-redis library. It is ok?